src/app/shared/enum/admins.ts

```diff
@@ -0,0 +1,12 @@
+export enum AdminRole {
+    all = 'all',
+    ministryAdmin = 'ministryAdmin',
+    regionAdmiin = 'regionAdmin',
+    territorialCommunityAdmin = 'territorialCommunityAdmin'
```

Comment:
I believe there is too much space, it should be 2 spaces

Reply:
fixed